src/commands/aksRotateClusterCert/aksRotateClusterCert.ts

```diff
@@ -0,0 +1,35 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { getAksClusterTreeItem, rotateClusterCert } from '../utils/clusters';
+import { failed, succeeded } from '../utils/errorable';
+import { longRunning } from '../utils/host';
+
+export default async function aksRotateClusterCert(
+  _context: IActionContext,
+  target: any
+): Promise<void> {
+  const cloudExplorer = await k8s.extension.cloudExplorer.v1;
+
+  const cluster = getAksClusterTreeItem(target, cloudExplorer);
+  if (failed(cluster)) {
+    vscode.window.showErrorMessage(cluster.error);
+    return;
+  }
+
+  const clusterName = cluster.result.name;
+
+  const answer = await vscode.window.showInformationMessage(`Do you want to rotate cluster ${clusterName} certificate?`, "Yes", "No");
+
+  if (answer === "Yes") {
```

Comment:
This is more of a personal preference, so feel free to disagree, but... I'd rather do:
```javascript
if (answer !== "Yes") {
    return;
}
const result = await longRunning(...);
```

That way, the 'main' code path doesn't get nested inside a conditional block, and it's easy to read where it exits early.